VaticanBAV/et/BAVet118.xml

```diff
@@ -201,11 +188,85 @@ schematypens="http://relaxng.org/ns/structure/1.0"
+                              
+                              <item xml:id="e1">
+                                  <note>
+                                      Many spaces left blank were filled in with Greek remedies and short texts.
+                                      On the following folios, the Greek text occupies a single column, leaving the other column blank: 
+                                      <locus from="30r" to="34v"/>, <locus from="38r" to="41v"/>, <locus from="44r" to="46v"/>, <locus from="49r" to="50r"/>,
+                                      <locus from="53r" to="54r"/>, <locus from="58r" to="60r"/>, <locus from="63r" to="64v"/>, <locus from="69r" to="70v"/>, <locus from="74r" to="76r"/>, <locus from="80r" to="81v"/>,
+                                      <locus from="86r" to="86v"/>, <locus from="91r" to="91v"/>, <locus from="96r" to="96v"/>, <locus from="102r" to="102v"/>, <locus from="108r" to="108v"/>, <locus from="113r" to="114r"/>,
+                                      <locus from="121r" to="121v"/>, <locus from="143r" to="143v"/>, <locus target="#161r"/>. <locus target="#150r"/>, 
+                                      On the following folios, the Greek text occupies the page in its entirety:
+                                      <locus from="1v" to="8r"/>, <locus from="20r" to="22v"/>, <locus from="23v" to="29v"/>, <locus target="#150v"/>, 
+                                      <locus from="161v" to="209r"/>, <locus from="224r" to="225v"/>, <locus from="228r" to="232r"/>.
+                                      The Greek texts begins as follows on <locus target="#2r"/>:
+                                      <q xml:lang="grc">Γραφομεν δηα δηαφορα γιατρικα ανακατομενα προτον δηα οάχα νοικύτ ηγουν κολπος ποὺντα του νερού κουρά μοιτζὲτ τα ρηζα 
+                                          οάγγηνα τα φύλα κόπανήζης όλα μαζή κάνης σκόνην ανακατεβης βούτηρο αληφης το κορλη του αστενους 7 ημερας
+                                      </q>
+                                  </note>
+                              </item>
+                              <item xml:id="e2">
+                                  <locus from="228r" to="230v"/>
+                                  <note>
+                                      Three square tables of magical or divinatory character, the first two of which are divided into 49 boxes (7x7) and the third into 64 boxes (8x8). 
+                                      Some symbols, such as a star, a hash or three or four vertical dashes, and other undecipherable symbols recur in the boxes of the three figures.
+                                      In the third figure on <locus target="#230r"/> these recurring symbols are repeated along the diagonal boxes.
+                                      The figures are accompanied by Greek text, starting as follows:
+                                      <q xml:lang="grc">ἡ παρούσα φιγούρα οφαιλη 18× ηδον πραξες μαιγας και τημηος ανθροπος</q>
```

Comment:
Is the numeral 18 written in the manuscript or is it 18 times this phrase?

Reply:
So appears in the Greek text printed by GrébTiss

Reply:
I do not know either, what this numeral and the symbol x stand for. You can perhaps mark it with `<sic resp="PRS4805Grebaut PRS9530Tisseran">`

![grafik](https://github.com/user-attachments/assets/5d168935-3c6c-4581-b9c1-381b140d9fb2)

Reply:
Thank you Carsten. I think I have solved this riddle: browsing in a b/w reproduction of BAVet118, it seems that this sign is a punctuation mark, because it appears also elsewhere, e.g. in the previous line after φιγούρα (something like X -- X). Perhaps it should be rendered as a full stop, because this is the intended use. What do you think?
![Screenshot (125)](https://github.com/user-attachments/assets/01aa0da8-6886-4f07-a17b-e1cbb809af4a)